src/main/java/io/cryostat/net/web/http/api/v1/TemplatesPostHandler.java

```diff
@@ -114,6 +115,11 @@ public boolean isAsync() {
         return false;
     }
 
+    @Override
```

Comment:
Is there a reason this is the only v1 endpoint that has content negotiation?

Reply:
No, I think I probably did that one by accident. There's no reason that v1 handlers can't or shouldn't also implement content negotiation, though. I'll back this one out for now and handle all the v1 ones together in a follow-up commit to this branch tomorrow.